src/modules/site-v2/templates/primary/map.html

```diff
@@ -0,0 +1,252 @@
+
+<div>
+        <div id="map" style="height: 500px; width:100%"></div>
```

Comment:
@orzut: we prefer moving the styles out of the code whenever possible. In this case, consider moving the height&width declarations to the existing <style/> section on the file, or if there is a references `.css` style file specific to this page, you can move this declaration there as well.